src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js

```diff
@@ -93,6 +93,12 @@ const Pagination = styled.ul`
   align-items: center;
 `;
 
+const PaperContainer = styled(Paper)`
```

Comment:
Not required. Don't make the padding 0 in mobile view. When you go to write a skill feedback you can see textfield touching the card border.